localnode/gen-files.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
```

Comment:
Ideally, I think it would be best if we could continue using `/bin/sh` and make the script POSIX compliant for compatibility on as many platforms as possible. I can do this if you want.